partner_contact_address_default/views/res_partner_views.xml

```diff
@@ -11,14 +11,12 @@
                         <field
                             name="partner_delivery_id"
                             domain="[('id', 'child_of', commercial_partner_id), ('type', '=', 'delivery')]"
-                            widget="selection"
```

Comment:
To keep the user experience, better use:
```xml
options="{'no_open': True, 'no_create': True}"
```

Reply:
Thanks.